app/webpacker/javascript/clarity.js

```diff
@@ -46,7 +46,10 @@ export default class Clarity {
   listenForConsentChanges() {
     document.addEventListener('cookies:accepted', () => {
       if (this.consentValue('non-functional') === 'granted') {
```

Comment:
Should we add another condition, so that if consent is later revoked, we inform Clarity not to track with:

```
window.clarity('consent', false)
```